pgxn/neon/neon--1.5--1.4.sql

```diff
@@ -0,0 +1,4 @@
+DROP VIEW IF EXISTS neon_perf_counters;
+DROP VIEW IF EXISTS neon_backend_perf_counters;
+DROP FUNCTION IF EXISTS get_perf_counters();
+DROP FUNCTION IF EXISTS get_backend_perf_counters();
```

Comment:
Are these accessed via `neon.get_*()`? Or are they naked?

Reply:
They are installed in the public schema, like other functions like `backpressure_lsn`, `neon_get_lfc_stats`, etc.

It probably would be good to put all these in a `neon` schema. But not going to change that in this PR...